README.md

```diff
@@ -1,32 +1,117 @@
+* Circuit Breaker Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-circuit-breaker-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-circuit-breaker-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-circuit-breaker) and [product](https://github.com/bucharest-gold/nodejs-circuit-breaker-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-circuit-breaker-booster) and [product](https://github.com/snowdrop/spring-boot-circuit-breaker-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker-redhat) boosters
+* Secured Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-rest-http-secured) and [product](https://github.com/bucharest-gold/nodejs-rest-http-secured-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-http-secured-booster) and [product](https://github.com/snowdrop/spring-boot-http-secured-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured-redhat) boosters
 
-## Setup
-```
+## Requirements
+For the first time, you have to install the [npm](https://www.npmjs.com/get-npm) and [protractor](http://www.protractortest.org).
+
+## List of boosters
+* http
+* configMap
+* crud
+* healthCheck
+* circuitBreaker
+* securedHttp
+
+## List of runtime
+* vertx
+* nodejs
+* springboot
+* wfswarm
+
+## Before run test
+### Install dependencies
+```bash
 $ npm install
-$ webdriver-manager update
-$ webdriver-manager start
 ```
-## Before Test
-Is necessary set up URL address for boosters, default value of config map and login credentials for secured booster
+### Update and run webdriver manager
+```bash
+$ npm run webdriver:start
+$ npm run webdriver:update
 ```
-export HTTP_BOOSTER_URL="url for http booster"
-export CRUD_BOOSTER_URL="url for crud booster"
-export CIRCUIT_BREAKER_BOOSTER_URL="url for circuit breaker booster"
-export CONFIG_MAP_BOOSTER_URL="url for config map booster"
-export HEALTH_CHECK_BOOSTER_URL="url for health check booster"
-export SECURED_HTTP_BOOSTER_URL="url for secured http booster"
 
-export CONFIG_MAP_BOOSTER_VALUE="Hello, %s from a ConfigMap !"
+## Run tests for all booster with the environmental variables
```

Comment:
I'm not interested in this. We can mention it towards the end of the README, but this is not the primary usecase.